vscode-lean4/src/utils/leanInstaller.ts

```diff
@@ -161,24 +156,18 @@ export class LeanInstaller implements Disposable {
             prompt += ` from ${path}`
         }
 
-        this.promptingInstallEmitter.fire(uri);
         if (shouldAutofocusOutput()) {
             this.outputChannel.show(true);
-        } else {
-            const outputItem = 'Go to output';
```

Comment:
this block was removed because it pops up an error message and blocks - this stops the real show install prompt message below on line 175, which we really don't want to block, that prompt takes precedence here.